### DISCUSSION TITLE
Choix de destination

### DISCUSSION THREAD

**Discussion by RayZZo9** [Crownicles/Crownicles#1915] [Suggestions fermées]
J'ai un problème et une question :

 -Mon problème :
Je sors de mergagnan, j'ai super besoin de partir vers la ville forte mais non, je m'en vais vers la valée des rois. (peut-être que je me trompe dans le nom des lieux mais c'est pas grave) Le truc, c'est que, en plus de me mettre un trajet plus long, cela me compromet dans la réalisation de mes missions, me rajoutant au moins 24h (ou +) de route. Même à part ça, en règle général, c'est ch...embêtant.

-Ma question :
Est-ce que on pourrait faire que je puisse à chaque fois choisir toutes les destinations possibles à prendre au lieu que ce soit le bot qui m'entraîne dans mon trajet pour la quête de la princesse (que je ne vois jamais d'ailleur) ? ça m'aiderais bcp.

**Comment by voltou91** [comment]
Hello,
Je sais que c'est frustrant mais je ne pense pas que ça bougera. Le fait que le bot puisse choisir quelques fois à ta place, rajoute de l'aléatoire et de l'imprévu à ton aventure. Je trouverais dommage si on l'enlevais.

**Reply by BastLast** [reply]
Je confirme
